### PR TITLE
[units] Update to 2.3.1

### PR DESCRIPTION
**Pull request by nairaner**
Update units to 2.3.1

unitsConfig.cmake is now installed using CMakeLists.txt

### CODE REVIEW

File: ports/units/portfile.cmake [hunk 1]
```diff
@@ -18,7 +18,6 @@ vcpkg_install_cmake()
 # Handle copyright/readme/package files
 file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/units RENAME copyright)
```
Comment (NancyLi1013) [inline ports/units/portfile.cmake:19]:
```suggestion
file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

File: ports/units/portfile.cmake [hunk 2]
```diff
@@ -18,7 +18,6 @@ vcpkg_install_cmake()
...
 file(INSTALL ${SOURCE_PATH}/README.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/units)
```
Comment (NancyLi1013) [inline ports/units/portfile.cmake:20]:
```suggestion
file(INSTALL ${SOURCE_PATH}/README.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]
`include(vcpkg_common_functions)` is no longer used.
Could you please remove this line?

Please also bump the version as` 2.3.0-1` in CONTROL file.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by strega-nil** [comment]
Alright, cool! Thanks @nairaner :)

**Pull request merged**